src/app/header/header.component.ts

```diff
@@ -32,7 +32,7 @@ export class HeaderComponent implements OnInit {
 
   myDashBoard(){
     console.log("load MyDashboard");
-    this.router.navigate(["/MyDashboard"])
+    this.router.navigate(["/"])
```

Comment:
Don't change the route here. This will create problem for going to MyDashboard component